README.md

```diff
@@ -42,6 +42,11 @@ To install the plugin, run:
   - `status <CR>` : Print the phase and conditions of CRs of a specific type, such as `cephobjectstore`, `cephfilesystem`, etc
   - `purge-osd <osd-id> [--force]` : Permanently remove an OSD from the cluster. Multiple OSDs can be removed with a comma-separated list of IDs.
 
+- `debug <subcommand>...`  : [Debug a deployment](#debug-mode)  by scaling it down and creating a debug copy. This is supported for mons and OSDs only"
```

Comment:
I just realized the other commands don't mention subcommands in this doc. This seems consistent
```suggestion
- `debug`  : [Debug a deployment](#debug-mode)  by scaling it down and creating a debug copy. This is supported for mons and OSDs only"
```

Reply:
Yup